### PR TITLE
Drop Python 2 support

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

**Comment by claudep** [comment]
Just refreshed by resolving the merge conflict.

**Comment by claudep** [comment]
Any reason why the checks are not run?

**Comment by moggers87** [comment]
I'll see if I can give travisci a prod

**Comment by moggers87** [comment]
Looks you've introduced an error into `.travis.yml`

File: .travis.yml [hunk 1]
```diff
@@ -13,11 +13,6 @@ env:
 
 matrix:
   include:
-    - python: 2.7
-      env: DJANGO=1.11 VARIANT=normal
-    - python: 2.7
-      env: DJANGO=1.11 VARIANT=yubikey
-    - python: 2.7
       env: DJANGO=1.11 VARIANT=custom_user
```
Comment (moggers87) [inline .travis.yml]: I think you missed line 16 when removing Python 2.7 tests from the matrix.

Comment (claudep) [reply .travis.yml]: Well spotted! I'll fix that.

**Comment by moggers87** [comment]
@Bouke I'm happy to merge this now if you are?

**Comment by Bouke** [comment]
Looks good to me!

**Pull request merged**